src/client/eppo-client.ts

```diff
@@ -835,6 +838,83 @@ export default class EppoClient {
     throw err;
   }
 
+  private getAllAssignments(
+    subjectKey: string,
+    subjectAttributes: Attributes = {},
+  ): Record<string, PrecomputedFlag> {
+    const configDetails = this.getConfigDetails();
+    const flagKeys = this.getFlagKeys();
+    const flags: Record<string, PrecomputedFlag> = {};
+
+    // Evaluate all the enabled flags for the user
+    flagKeys.forEach((flagKey) => {
+      const flag = this.getFlag(flagKey);
+      if (flag === null || !flag.enabled) {
+        logger.info(`[Eppo SDK] No assigned variation. Flag is disabled: ${flagKey}`);
+        return;
+      }
+
```

Comment:
To simplify the code: this check is already included in the evaluator and it returns a `noneResult` - https://github.com/Eppo-exp/js-sdk-common/blob/main/src/evaluator.ts#L61

Is it possible to result that and filter afterwards, which you're already doing on line 867

Possible checking for a defined flag is still needed

```suggestion
```

Reply:
still need to do a null check so I've left that in.